lib/misc.c

```diff
@@ -1282,3 +1282,35 @@ gftp_get_share_dir (void)
   return (gftp_share_dir);
 }
 
+char *
+gftp_format_file_size(off_t bytes) {
+  char tmp[20];
+  char ret[40];
+  char *pointzero = 0;
+  if (bytes >= 1099511627776) {
```

Comment:
What do you think about writing this as `if (bytes >= 1024*1024*1024*1024) {`? I think it's a little easier to read and audit.